### PR TITLE
Create info.ini

### PR DESCRIPTION
**Pull request by dominicrico**

### CODE REVIEW

**Comment by megawac** [comment]
Hey, you're going to have to add some documentation to your project before we can host it. Also it may be better to just host the minified files?

**Comment by dominicrico** [comment]
Hey, yes for sure! I will commit it in a second.

File: files/angular-reading-indicator/update.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "angular-reading-indicator",
+  "repo": "dominicrico/angular-reading-indicator",
+  "files": {
+    "include": ["src/ng-reading-indicator.js", "dist/ng-reading-indicator.min.js", "dist/ng-reading-indicator.min.js.map", "src/ng-reading-indicator.css", "dist/ng-reading-indicator.min.css"]
```
Comment (megawac) [inline files/angular-reading-indicator/update.json]: Can you add a `basePath: dist/` and remove src/ files?
Thanks

Comment (dominicrico) [inline files/angular-reading-indicator/update.json]: Added with last commits

**Comment by jimaek** [comment]
Delete `/README.MD` from the PR please

**Comment by dominicrico** [comment]
@megawac said before that I have to add some documentation, but I will delete it.

**Comment by jimaek** [comment]
Rename the directory to `angular.reading-indicator` and you are good to go

**Comment by megawac** [comment]
@dominicrico I meant on your github repo ;)

**Comment by dominicrico** [comment]
Haha :smile:  ok, I was doing this while committing here.

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Pull request closed without merging**